src/compiler/diagnosticMessages.json

```diff
@@ -3536,6 +3536,14 @@
         "category": "Error",
         "code": 7036
     },
+    "Enables emit interoperability bewteen commonjs and ES Modules via creation of namespace objects for all imports. Implies `allowSyntheticDefaultImports`.": {

```

Comment:
**between**

**CommonJS**

Stick with consistent inner quote marks - we usually use apostrophes/single-quotes.